Makefile

```diff
@@ -0,0 +1,6 @@
+# LightStep-specific: rebuilds the LightStep thrift protocol files.  Assumes
```

Comment:
I think (technically) you also want to put a 

```make
.PHONY: thrift
```

somewhere before the rule so Make knows `thrift` is a pure target, not a file that's being generated.